src/sentry/monitors/rate_limit.py

```diff
@@ -0,0 +1,84 @@
+from typing import Type
+
+from django.core.cache import cache
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from sentry.models.project import Project
+from sentry.monitors.models import MonitorEnvironment
+from sentry.tasks.relay import schedule_invalidate_project_config
+
+# Determines how many check-ins per-minute will be allowed per monitor. This is
+# used when computing the QuotaConfig for the DataCategory.MONITOR (check-ins)
+#
+#
+# Each monitor should __really__ only be checking-in once per minute, but we
+# will give some lee-way to allow for customers whos monitors check-in early.
+ALLOWED_CHECK_INS_PER_MONITOR_PER_MINUTE = 5
+
+# The minimum computed project_monitor_rate_limit. This value should be high
+# enough that it allows for a large number of monitors to be upserted without
+# hitting the project rate-limit.
+ALLOWED_MINIMUM = 50
```

Comment:
This effectively means we are starting you out as if you have 10 monitors already created.

In effect this means you can create 50 monitors via upsert per minute.

Reply:
is there any way we can tie this to billing seats? or would that not make sense?

Reply:
Yeah we probably can. But unclear what would make sense there.